packages/Form/core/src/FieldForm.js

```diff
@@ -83,14 +82,17 @@ class FieldForm extends Component {
     };
   }
 
-  componentWillReceiveProps(nextProps) {
-    const { forceDisplayMessage } = this.props;
-    if (forceDisplayMessage !== nextProps.forceDisplayMessage) {
-      const memory = this.captureMemory();
-      this.setState({
-        memory,
-      });
+  static getDerivedStateFromProps(props, state) {
+    if (props.forceDisplayMessage !== state.forceDisplayMessage) {
```

Comment:
Stage.forceDisplayMessage does need or seem to be in the state.
Are you sure ti have the same behavior Xavier?

Reply:
Yes you're right it is not in the state. However what is the expected behavior and how can I test it?

Reply:
Fonctionnellement, l'idée est que la gestion du comportement des messages d'erreurs est gérée automatiquement (focus, ect). 

Le comportement d'affichage des messages d'erreurs est dissocié de la validation des messages d'erreures.
- Le: es ce qu'un champ est en erreurs est généré par l'application
- Le: comment s'affiche mes champs est déjà prégéré par le design system


Il y a des use case ou l'on veut reprendre la main. Par exemple à l'arrivée sur un formulaire, je veux déjà pré-afficher les champs en erreurs, d'ou le "forceDisplayMessage" qui permet de ne plus utiliser le state interne, mais les forcer l'utilisation des information provenant des props et de ne plus utiliser le state interne.

Reply:
Comme vu ensemble, j'ai supprimé getDerivedStateFromProps car il ne servait à rien en fait.